### ISSUE TITLE
Commands for 1000 epoch training

### ISSUE THREAD

**Issue by jlindsey15** [google-research/simclr#56]
Hi!  Were the hyperparameters used for the highest-performing, 1000-epoch trained SimCLR network exactly the same as those given in the repo for 100 epochs?  I.e. is reproducing those results as simple as changing the epochs flag to 1000, or are other changes needed?  Thanks!

**Comment by chentingpc** [comment]
Yes, they should be the same. But I haven't tested it on cloud tpu setting for long epoch training, so the results may differ. I suggest you run with different epochs (e.g. 100, 200), and check if the results match that of Table B.1 in the appendix of paper (where we listed our results on different bsz and training epochs under two learning rate scaling rules).

**Comment by jlindsey15** [comment]
Thanks!
